### PR TITLE
Implement play_file_ext with offset parameter in ms

### PR DESCRIPTION
**Pull request by larsimmisch**
This PR allows to start playing a file with an offset, which is useful for resuming audio where it was interrupted

### CODE REVIEW

File: src/play.c [hunk 1]
```diff
@@ -242,7 +255,7 @@ static int aufile_load(struct mbuf *mb, const char *filename,
 	mem_deref(af);
 
 	if (!err) {
-		mb->pos = 0;
+		mb->pos = (prm.srate * fsize * prm.channels / 1000) / file_size;
```
Comment (sreimers) [inline src/play.c:258]: I think you can use `aufmt_sample_size(prm.fmt)` instead `fsize` here.

File: include/baresip.h [hunk 2]
```diff
@@ -784,6 +788,8 @@ int  play_init(struct player **playerp);
 void play_set_path(struct player *player, const char *path);
 
 
+int enqueue(const char* molecule, void* arg);
```
Comment (sreimers) [inline include/baresip.h:791]: This looks unrelated?

File: src/play.c (hunk 1 above)
Comment (larsimmisch) [reply src/play.c]: Yes, that's much short and more complete (done)

File: include/baresip.h (hunk 2 above)
Comment (larsimmisch) [reply include/baresip.h]: Uh, that needs to go away.

File: src/play.c [hunk 3]
```diff
@@ -207,42 +218,12 @@ static int aufile_load(struct mbuf *mb, const char *filename,
...
-			for (i=0; i<n; i++) {
-				err |= mbuf_write_u16(mb,
-						      g711_ulaw2pcm(buf[i]));
-			}
-			break;
-
-		default:
-			err = ENOSYS;
-			break;
-		}
```
Comment (sreimers) [inline src/play.c:239]: deleted by mistake?

File: src/play.c [hunk 4]
```diff
@@ -207,42 +218,12 @@ static int aufile_load(struct mbuf *mb, const char *filename,
...
 	}
 
 	mem_deref(af);
 
 	if (!err) {
-		mb->pos = 0;
+		mb->pos = (prm.srate * aufmt_sample_size(prm.fmt) * prm.channels / 1000) / file_size;
```
Comment (sreimers) [inline src/play.c:226]: Should `play->offset_ms` involved here?

Comment (larsimmisch) [reply src/play.c:226]: Yes - that part is missing entirely and I think I'll have to prepare another PR for `aufile_set_pos` in re. I'll close this PR for now.
